backend/controllers/userController.js

```diff
@@ -1,7 +1,6 @@
 const bcrypt = require('bcryptjs');
 const jwt = require('jsonwebtoken');
 const User = require('../models/userModel');
-const CustomError = require("../utils/CustomError.js");
```

Comment:
Sir, while resolving merge conflicts, this import of CustomError was made redundant and caused an error. So with this pr, I removed it. That's why I have added this file intentionally